dazel.py

```diff
@@ -425,6 +477,13 @@ def _add_volumes(self, volumes):
         # Calculate the volumes string.
         self.volumes = '-v "%s"' % '" -v "'.join(volumes)
 
+    def _get_remote_directory(self, local_directory):
+        remote_directory = local_directory
+        if (sys.platform == "win32"):
+            win_path = os.path.splitdrive(local_directory)[1]
+            remote_directory = str(pathlib.PureWindowsPath(win_path).as_posix())
```

Comment:
Resolved in README file since this applies to DAZEL_VOLUMES

Reply:
I might be missing something, but it seems like (for example) `self.remote_directory`, which is defined in the `__init__` method, will not have a drive letter folder inside docker...

Is that on purpose?

Reply:
I didn't see it was necessary and I didn't want to add it to all directories since it creates problems with .cache. I have added it now to only self.remote_directory.